src/libs/ReportUtils.js

```diff
@@ -602,7 +602,7 @@ function getOldDotDefaultAvatar(login = '') {
  * @returns {Boolean}
  */
 function isDefaultAvatar(avatarURL) {
-    if (_.isString(avatarURL) && (avatarURL.includes('images/avatars/default-avatar_') || avatarURL.includes('images/avatars/user/default'))) {
+    if (_.isString(avatarURL) && (avatarURL.includes('images/avatars/avatar_') || avatarURL.includes('images/avatars/default-avatar_') || avatarURL.includes('images/avatars/user/default'))) {
```

Comment:
@francoisl to fix linting
```suggestion
    if (_.isString(avatarURL) && (
        avatarURL.includes('images/avatars/avatar_')
        || avatarURL.includes('images/avatars/default-avatar_')
        || avatarURL.includes('images/avatars/user/default'))
    ) {
```

Reply:
Ah, sorry Github didn't notify me of your suggestion before I pushed my update. I went for something a little different, but we can use your suggestion if you think it's better?